### PR TITLE
[AAE-18175] Unable to Import Descriptor in HXP Environment

### PR DESCRIPTION
**Pull request by BSekula**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://alfresco.atlassian.net/browse/AAE-18175

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/src/lib/auth/authentication-interceptor/auth-bearer.interceptor.ts [hunk 1]
```diff
@@ -72,7 +72,7 @@ export class AuthBearerInterceptor implements HttpInterceptor {
     // prevent adding any content type, to properly handle formData with boundary browser generated value,
     // as adding any Content-Type its going to break the upload functionality
 
-    if (headers.get('Content-Type') === 'multipart/form-data') {
+    if (headers.get('Content-Type') === 'multipart/form-data' && !(reqBody instanceof FormData)) {
```
Comment (alep85) [inline lib/core/src/lib/auth/authentication-interceptor/auth-bearer.interceptor.ts:75]: Did you try if it still work on apa?

Comment (BSekula) [reply lib/core/src/lib/auth/authentication-interceptor/auth-bearer.interceptor.ts:75]: nop, I am going to test it on apa as well 👍

Comment (alep85) [reply lib/core/src/lib/auth/authentication-interceptor/auth-bearer.interceptor.ts:75]: thanks 🙏  so we're sure that works on both

Comment (BSekula) [reply lib/core/src/lib/auth/authentication-interceptor/auth-bearer.interceptor.ts:75]: Tested all apps with adf configuration, looks good on local:

- workspace hxp - OK
- content-ee-apa - OK
- studio-hxp - OK
- modeline-ee - OK
- admin-hxp - OK
- admin-apa - OK

**Review by DudaRobert — APPROVED** [review]

**Pull request merged**